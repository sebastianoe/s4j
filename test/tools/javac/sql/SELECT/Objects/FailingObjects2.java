/*
 * @test  (c) SO
 * @summary Fail due to unassignable projection
 * @compile businessobjects/Product.java
 * @compile businessobjects/Storage.java
 * @compile/fail/ref=FailingObjects2.out FailingObjects2.java -sqlrepo /Users/sebastian/ma/jdk7u-langtools/test/tools/javac/sql/repo.json
 */

import businessobjects.Product;
import businessobjects.Storage;

public class FailingObjects2 {
    public static void main(String[] args) {
    	Product p = SQL[SELECT * FROM customer];
    }    
}