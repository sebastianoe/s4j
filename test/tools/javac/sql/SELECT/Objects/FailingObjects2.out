error: The mapping contains no-preference pairs. Cannot assign projections to attributes.
1 error